Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -868,6 +868,50 @@ def identityFS(wf):
 steps['RECOHID10']['--datatier']+=',RAW'
 steps['RECOHID10']['--eventcontent']+=',REPACKRAW'
 
+steps['RECOHID12']=merge([{'--scenario':'HeavyIons',
```

Comment:
Hi @jrcastle adding several new steps to test different AlCaRecos is not needed.
Moreover all this `RECOHID1`' are not used in any workfow of `relval_standard.py`. The only workflows that are used for HI are: 140.51,140.52,140.53 see [here](https://github.com/jrcastle/cmssw/blob/HeavyIonCommonAlignmentProducer_75X/Configuration/PyReleaseValidation/python/relval_standard.py#L106).
My suggestion is to put everything in the same step (i.e. `steps['RECOHID10']`) by adding:
```
'-s':'RAW2DIGI,L1Reco,RECO,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBiasHI+TkAlUpsilonMuMuHI+TkAlZMuMuHI+TkAlMuonIsolatedHI+HcalCalMinBias,DQM',
```
Once this is done we can probe the new producers directly in the PR since in the IB tests 150.53 is run.
Please let us know.